docs/en/latest/configuration/config-yaml.md

```diff
@@ -0,0 +1,556 @@
+            <tr>
+                <td>string</td>
+                <td>&quot;&quot;</td>
+                <td>Path of your self-signed CA certificate. The CA is used to sign all admin api callers certificates.
+                </td>
+            </tr>
+            <tr>
+                <td rowspan=3><code>admin_listen
+            <ul><li>ip</li>
+            <li>port</li>
+            </ul>
+            </code>
+                </td>
+                <td></td>
+                <td></td>
+                <td>Use a separate configuration for admin to listen on.</td>
+                <td rowspan=3>Disabled by default</td>
+            </tr>
+            <tr>
+                <td>IP address</td>
+                <td>127.0.0.1</td>
+                <td>Specific IP, if not set, then the default value is 0.0.0.0</td>
+            </tr>
+            <tr>
+                <td>port</td>
+                <td>9180</td>
+                <td>Specific port</td>
+            </tr>
+            <tr>
+                <td><code>allow_admin</code></td>
+                <td>IP list</td>
+                <td>127.0.0.0/24</td>
+                <td>Allow admins only on specified addresses. Allows on all addresses when not IP list is set.</td>
+                <td>Put valid IP addresses only. For IPv6 <code>::/64</code>.</td>
+            </tr>
+            <tr>
+                <td><code>config_center</code></td>
+                <td>string</td>
+                <td>etcd</td>
+                <td>Use config from yaml file or store config to etcd.</td>
+                <td>Possible values: <em>etcd</em> or <em>yaml</em></td>
+            </tr>
+            <tr>
+                <td rowspan=3>
+                    <code>control
+                        <ul>
+                            <li>ip</li>
+                            <li>port</li>
+                        </ul>
+                    </code>
+                </td>
+                <td colspan=3></td>
+                <td rowspan=3>Disabled by default</td>
+            </tr>
+            <tr>
+                <td>IP address</td>
+                <td>127.0.0.1</td>
+                <td></td>
+            </tr>
+            <tr>
+                <td>port</td>
+                <td>9090</td>
+                <td></td>
+            </tr>
+            <tr>
+                <td><code>delete_uri_tail_slash</code></td>
+                <td>boolean</td>
+                <td>false</td>
+                <td>Enabling this will remove any terminating <strong>/</strong> from the end of the URI.</td>
+                <td></td>
+            </tr>
+            <tr>
+                <td><code>disable_sync_configuration_during_start</code></td>
+                <td>boolean</td>
+                <td>false</td>
+                <td></td>
+                <td></td>
+            </tr>
+            <tr>
+                <td><code>dns_resolver</code></td>
+                <td>IP addresses</td>
+                <td>1.1.1.1, 8.8.8.8</td>
```

Comment:
`1.1.1.1` is not a DNS address.

Reply:
It is Cloudflare DNS service just like Google DNS: https://www.cloudflare.com/learning/dns/what-is-1.1.1.1/